cloud-nio/cloud-nio-impl-drs/src/main/scala/cloud/nio/impl/drs/DrsCloudNioFileSystemProvider.scala

```diff
@@ -35,8 +35,11 @@ class DrsCloudNioFileSystemProvider(rootConfig: Config,
 
   override def getScheme: String = "dos"
```

Comment:
Changes to accept `drs://` scheme also need to happen here as it is being used in other places, one of them being in `DrsPathBuilder` for scheme validation using ['drsScheme'](https://github.com/broadinstitute/cromwell/blob/957db2072c6202bf991e66d5d98e24f5c5851515/filesystems/drs/src/main/scala/cromwell/filesystems/drs/DrsPathBuilder.scala#L15) variable.